kalite/distributed/templates/distributed/base.html

```diff
@@ -82,9 +84,26 @@
 
         <script type="text/javascript" src="{% static 'js/fastclick.js' %}"></script>
         <script>
+
         $(function() {
             FastClick.attach(document.body);
         });
+
+         /* TODO(jtamiace): Backbonify this function and this page... when the time comes.
```

Comment:
@rtibbles Definitely will tackle this when I'm not working on content curation; great chance to practice backbone. There's a couple of months left in this cycle right?